exercises/darts/canonical-data.json

```diff
@@ -0,0 +1,65 @@
+{
+  "exercise": "darts",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Return the correct amount earned by a dart landing in a given point in the target problem.",
+      "cases": [
+        {
+         "property": "result",
+         "description": "A dart lands outside the target",
+         "input": {
+           "x": 15.3,
+           "y": 13.2
+         },
+         "expected": 0
+        },
+        {
+         "property": "result",
+         "description": "A dart lands just in the border of the target",
+         "input": {
+           "x": 10,
+           "y": 0
+         },
+         "expected": 1
+        },
+        {
+         "property": "result",
+         "description": "Input is not a number",
```

Comment:
I was thinking that any invalid input should return `null`. Maybe I should make it clear in the description, or go with the error. What do you think?

Reply:
The convention is that an error condition should be formatted as follows: `"expected": {"error": "empty stack"}`
Each track implementing this exercise can then for itself decide how to represent the error case.

Reply:
I think the case should be excluded from the exercise.  See #902 and #523.

Reply:
@ramadis What do you think about excluding this (error) case from the exercise? @rpottsoh has linked to two relevant issues.

Reply:
@ErikSchierboom  Yes, I'm inclined to go that way. I believe also that adding tests for inputs that are outside the problem domain in the problem specification could bring inconsistencies between implementations. Since, for example, a language with a strong type system (e.g. Haskell) shouldn't have to test for these cases since the program would not compile given incorrect input, and making it throw an exception would be counterproductive to the simplicity and core value of the problem.

So, I'll remove this test and keep going on!